src/index.js

```diff
@@ -1,6 +1,5 @@
 import $ from 'jquery';
 import './css/base.scss';
-import User from './User';
```

Comment:
Looks like because you are importing User into your FilterData class you removed it here to remove the infinite loop of death.